CHANGELOG

```diff
@@ -1,3 +1,10 @@
+Version 0.6.1 (2016-02-17)
+--------------------------
+Made podfile dependencies more restrictive (#266)
+Harmonized fg/bg/session check intervals with Android (#265)
```

Comment:
I think the session check is unchanged, so perhaps we can rename ticket & commit to "Harmonize foreground and background timeouts with Android"...